.travis.yml

```diff
@@ -0,0 +1,45 @@
+language: python
+cache: pip
+
+env:
+  global:
+    - PYTHONPATH="./tests/"
+    - DJANGO_SETTINGS_MODULE="settings_test"
+
+matrix:
+  include:
+    - python: 3.6
+    # Workaround for Python 3.7 support. See:
+    # https://github.com/travis-ci/travis-ci/issues/9815
+    - python: 3.7
+      dist: xenial
+      sudo: true
+
+# Workaround for testing Boto3 on Travis. Taken from:
+# https://github.com/azavea/django-amazon-ses/blob/57ed8619435194742308c6f15cbf9c91b231b88a/.travis.yml#L17-L19
+before_install: sudo rm /etc/boto.cfg
+
+install: pip install .[tests]
+
+script: django-admin test --noinput
+
+deploy:
+  - provider: pypi
+    user: django-ecsmanage
```

Comment:
I created the `django-ecsmanage` user to be a collaborator on the PyPi project, and I stored the credentials in LastPass. I was unable to get the `azavea` user to authenticate properly due to known problems with special characters in PyPi credentials. See: https://docs.travis-ci.com/user/deployment/pypi/